packages/deployment/lib/lambda.js

```diff
@@ -42,18 +41,19 @@ class UpdatedLambda extends Lambda {
    */
 
   async zipLambda(lambda) {
-    let msg = `Zipping ${lambda.local}`;
     // skip if the file with the same hash is zipped
     // and is a valid zip file
-    if (fs.existsSync(lambda.local)) {
+    if (await fs.pathExists(lambda.local)) {
```

Comment:
Use an asynchronous operation instead of hanging the process on the `fs.existsSync` call.